packages/ceramic-cli/src/ceramic-client.ts

```diff
@@ -65,7 +66,7 @@ class Document extends EventEmitter {
   }
 
   async change (newContent: any, apiUrl: string): Promise<boolean> {
-    const { docId, state } = await fetchJson(apiUrl + '/change' + this.id, { content: newContent })
+    const { state } = await fetchJson(apiUrl + '/change' + this.id, { content: newContent })
```

Comment:
Let's normalize the URL concatenation in order to avoid multiple slashes if the `apiUrl` already ends with `/`. Maybe a separate utility function.